scripts/release.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
-set -euo pipefail
 
-python manage.py migrate --noinput
-python manage.py createcachetable
-python manage.py import_shapes data/final/boundary.geojson
+if [ -n "${PRODUCTION}" ] || [ -n "${STAGING}" ]; then
+    set -euo pipefail
+
+    python manage.py migrate --noinput
+    python manage.py createcachetable
+    python manage.py import_shapes data/final/boundary.geojson
+    python manage.py clear_cache
```

Comment:
this line seems new. i wonder why the cache wasn't getting cleared before. seems fine to include now